xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/job/AbstractEntityJobWithChecks.java

```diff
@@ -63,13 +64,28 @@ protected void runInternal() throws Exception
                 // Process
                 progressManager.startStep(this);
                 setContextUser();
+
+                // FIXME: this should probably be the selected entities only...
```

Comment:
Do you still plan to address this?

Reply:
That's what I started to check as part of https://github.com/xwiki/xwiki-platform/pull/3887. I'm not planning to handle this as part of the current PR at least.